index.html

```diff
@@ -0,0 +1,184 @@
+        </div>
+      </div>
+      <!-- Card-2 -->
+      <div class="card">
+        <div class="card-image">
+          <i class="fas fa-clock"></i>
+        </div>
+        <div class="card-header">
+          OPEN 7 DAYS A WEEK
+        </div>
+        <div class="card-body">
+          <p>Flexible hours to meet everyone's scheduling needs.</p>
+        </div>
+      </div>
+      <!-- Card-3 -->
+      <div class="card">
+        <div class="card-image">
+          <i class="fa-sharp fa-solid fa-hand-sparkles"></i>
+        </div>
+        <div class="card-header">
+          CLEAN, SPACIOUS FACILITIE
+        </div>
+        <div class="card-body">
+          <p>From front lobby and fitness machines to the bathrooms , a clean gym is one of our top priorities.</p>
+        </div>
+      </div>
+    </div>
+    <a href="#" class="btn-join">Join GOLD GYM</a>
+  </section>
+
+  <!-- Feature Speaker -->
+  <section class="services feature-speaker">
+    <h1 class="services-header speaker-header">Clients Feedback</h1>
+    <div class="f-card-wrapper change-height">
+      <!-- Card-1 -->
+      <div class="card">
+        <div class="card-image">
+          <img src="source/images/jay-cutler.jpg" alt="">
+        </div>
+        <div class="card-header" id="card-header-1">
+        </div>
+        <div class="card-body">
+          <p id="card-body-1"></p>
+        </div>
+      </div>
+      <!-- Card-2 -->
+      <div class="card">
+        <div class="card-image">
+          <img src="source/images/andri.jpg" alt="">
+        </div>
+        <div class="card-header" id="card-header-2">
+        </div>
+        <div class="card-body">
+          <p id="card-body-2"></p>
+        </div>
+      </div>
+      <!-- Card-3 -->
+      <div class="card">
+        <div class="card-image">
+          <img src="source/images/body.jpg" alt="">
+        </div>
+        <div class="card-header" id="card-header-3">
+        </div>
+        <div class="card-body">
+          <p id="card-body-3"></p>
+        </div>
+      </div>
+      <!-- Card-4 -->
+      <div class="card">
+        <div class="card-image">
+          <img src="source/images/women2.jpg" alt="">
+        </div>
+        <div class="card-header" id="card-header-4">
+        </div>
+        <div class="card-body">
+          <p id="card-body-4"></p>
+        </div>
+      </div>
+      <!-- Card-5 -->
+      <div class="card">
+        <div class="card-image">
+          <img src="source/images/women1.jpg" alt="">
+        </div>
+        <div class="card-header" id="card-header-5">
+
+        </div>
+        <div class="card-body">
+          <p id="card-body-5"></p>
+        </div>
+      </div>
+    </div>
+    <button class="btn-join" id="more">More <i class="fa-sharp fa-solid fa-angle-down"></i></button>
+  </section>
+
+  <!-- Partener -->
+  <section class="partner">
+    <h1 class="p-header">Partner</h1>
```

Comment:
- [ ] There should be no more than 1 `<h1>` tag on a page. Please take a look at [this article](https://www.contentkingapp.com/academy/headings/) about the headings and fix this issue.